mainui/settings/persistence.md

```diff
@@ -9,21 +9,26 @@ title: Settings - Persistence
 **Persistence** stores data over time.
 The data may be retrieved at a later time, for example to restore your system after startup, or to prepare graphs for display on the UI.
 
-openHAB ships with the [RRD4J persistence service](/addons/persistence/rrd4j/) installed by default, which persists every Item on every state change and at least once a minute.
-Additionally, it restores the last stored value at system startup.
+openHAB ships with the [RRD4J persistence service](/addons/persistence/rrd4j/), which by default:
```

Comment:
Actually, OH doesn't ship with *any* persistence anymore. The first run wizard strongly suggests users to install it but you can refuse. Given the additions below I think this whole sentence with the bullet points can be deleted.